docs/src/calibration_features.md

```diff
@@ -1,28 +1,59 @@
+- **Statistics**: The statistical methods that can be applied on the data and simulation results to compare them.
+
+## How to run a calibration
+To be able to run a calibration, you need to import the `/KinematicDriver/CalibrateCMP` folder. 
+## Example
+To better understand how to build the configuration file and run calibrations in KiD, you can find a 1-dimensional calibration example in the `test/experiments/calibrations` folder.  In this example, we calibrate against the PySDM products of rain water mixing ratio (``r_r``) and cloud water mixing ratio (``r_l``), and KiD uses the [1-moment microphysics bulk scheme](https://clima.github.io/CloudMicrophysics.jl/dev/Microphysics1M/) to compute the same products. The parameters that are calibrated are ``\chi_v``, and ``\chi_a``, two parameters that appear in the terminal velocity (``v_{term}(r)``) and rain cross-section (``a(r)``) formulas of the bulk scheme:
+```math
+a(r) = \chi_a \, a_0 \left(\frac{r}{r_0}\right)^{a_e + \Delta_a}
+```
+```math
+v_{term}(r) = \chi_v \, v_0 \left(\frac{r}{r_0}\right)^{v_e + \Delta_v}.
+```
+
+Inside the calibration example, there are examples of the `config.jl` file, the `run_calibration.jl` file, and `run_model.jl`.
+
+In `config.jl`, we define the function `get_config()`, which creates the configuration dictionary. The keys of this dictionary are associated with all the elements that are needed for the calibration process, and they are retrieved using the following functions:
+- `get_prior_config()`: Sets up the priors' configuration.
+- `get_observations_config()`: Specifies the reference data.  
+- `get_process_config()`: Defines the method for the calibration and the associated parameters.
+- `get_model_config()`: Defines the model setup.
+- `get_stats_config()`: Specifies the statistical methods used on the reference data and simulation results.
+
+After creating the configuration file, running the calibrations using the `run_calibration.jl` file becomes very straightforward. 
+By importing the `/KinematicDriver/src/CalibrateCMP` folder and including the configuration file, you can use the appropriate functions to construct the priors and get the reference observations. Depending on the calibration method that you choose, the `calibrate()` and `get_results()` functions allow you to perform the calibration steps and obtain the final set of optimal parameters. 
+When choosing the EKI or UKI methods, `calibrate()` outputs the new updated ensemble and the optimal parameters for the calibration problem, that are given by the means of the last ensembles. To obtain the optimal values in the constrained space as well as the covariance connected to the distribution that describes the solution of the inversion problem, you can simply call the `get_results()` function.
```

Comment:
.... outputs the posterior ensemble and the optimal parameters, that are computed based on the final ensemble of parameters (statistical mean).